.travis.yml

```diff
@@ -29,11 +35,12 @@ before_install:
   - git-ensembl --clone --depth 1 --branch $ENSEMBL_BRANCH ensembl-io
   - git-ensembl --clone --depth 1 --branch $ENSEMBL_BRANCH ensembl-variation
   - git-ensembl --clone --depth 1 --branch $ENSEMBL_BRANCH ensembl-compara
-  - git clone -b release-1-6-924 --depth 1 https://github.com/bioperl/bioperl-live.git
 
 install:
-  - cpanm -v --sudo --notest XML::LibXML # https://github.com/shlomif/perl-XML-LibXML/pull/87
-  - cpanm -v --sudo --installdeps --with-all-features .
+  # Known test failures with new libxml2: https://github.com/shlomif/perl-XML-LibXML/pull/87
+  - cpanm -v --sudo --notest XML::LibXML
+  # Tests fail for Bio-DB-NCBIHelper, so use --notest
```

Comment:
Did not check ... any idea why Bio::DB::NCBIHelper fails? Is it a bug in its test suit - similarly to XML::LibXML?